.github/workflows/test-run.yml

```diff
@@ -0,0 +1,29 @@
+name: Test Run
+
+on: [push, pull_request]
+
+
+jobs:
+  test-run:
+    runs-on: ubuntu-18.04
+
+    steps:
+      - name: Checkout repository files
+        uses: actions/checkout@v2
+
+      - name: Edit Makefile
+        run: |
+          sed -i 's/^NUM_GPUS=8/NUM_GPUS=0/' Makefile
+          sed -i 's/^SHARED_ROOT.*/SHARED_ROOT=\/tmp/' Makefile
+
+      - name: Install necessary dependencies
+        run: |
+          make conda
+          make snakemake
+          make git-modules
+
+      - name: Dry run
+        run: make dry-run
+
+      - name: Test run
```

Comment:
So, I suggest removing the test step for now and merging the dry-run. 

We can continue working on the test step in a separate PR. It will require including a GPU machine in the setup and outputting or saving logs of failed jobs to investigate issues. + experimenting with the test config to make it run faster